.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "daily"
+    time: "11:00"
+  open-pull-requests-limit: 10
+  assignees:
+    - "dependabot"
+
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "11:00"
+  open-pull-requests-limit: 10
```

Comment:
```suggestion
  open-pull-requests-limit: 5
```

Reply:
Looks like 5 is the default, according to the docs, so we can just remove this

> To keep pull requests manageable and easy to review, Dependabot raises a maximum of five pull requests to start bringing dependencies up to the latest version.

From the docs: https://docs.github.com/en/github/administering-a-repository/enabling-and-disabling-version-updates#checking-the-status-of-version-updates